### DISCUSSION TITLE
using balance datapoint

### DISCUSSION THREAD

**Discussion by kpnobvious** [jertel/vuegraf#82] [Q&A] [answered]
HI,
I see back on Sept 1 you added an update for balance and total usage.

I see it in the release code
def extractDataPoints(device, usageDataPoints):
    excludedDetailChannelNumbers = ['Balance', 'TotalUsage']

But, i don't see it showing up in the channel extract and i would guess I would also have to add it to my json file.  Is that the right way to get this point added?

**Comment by jertel** [comment, marked as answer]
No, you don't need to do anything for the Balance metric to show up. It will be entered into the InfluxDB as `<Panel Name>-Balance`. Ex: `Right Panel-Balance`.

TotalUsage will only appear if you have Solar configured in your Vue system.

**Reply by kpnobvious** [reply]
ok, its not showing up.  I thought i would see it get calculated in vuegraf.py.  Does it log as a discovered channel or depend on something from pyemvue to get the balance?  I also thought if it was not defined in the json file it would get dropped

**Reply by jertel** [reply]
No, it does not log as a discovered channel, and no, it does not need to be defined in the config file. Provide some screenshots and/or influxDB query/results so I can help.

**Reply by kpnobvious** [reply]
thanks!  Definitely some weird issue locally.  I cloned a fresh copy from git and pointed it to a different database with my current json.  It's logging 18 data points a minute including <Panel Name>-Balance in the influxDB.  My current instance is only logging 17 points and no Balance.
